src/python/pants/backend/python/tasks2/python_execution_task_base.py

```diff
@@ -101,7 +104,8 @@ def extra_requirements(self):
   def create_pex(self, pex_info=None):
     """Returns a wrapped pex that "merges" the other pexes via PEX_PATH."""
     relevant_targets = self.context.targets(
-      lambda tgt: isinstance(tgt, (PythonRequirementLibrary, PythonTarget, Files)))
+      lambda tgt: isinstance(tgt, (PythonDistribution,
```

Comment:
indentation here is valid but a bit odd.  I think we generally prefer to break earlier. 
```
lambda tgt: isinstance(tgt, (
  PythonDidstribution, ...))
```
or 
```
lambda tgt: isinstance(
  tgt,
  (PythonDistribution...)
)
```
seems to be a bit better.

Reply:
✅